pkg/flags/flags.go

```diff
@@ -228,14 +228,19 @@ https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-g
 	flags.IntVar(&nginx.MaxmindRetriesCount, "maxmind-retries-count", 1, "Number of attempts to download the GeoIP DB.")
 	flags.DurationVar(&nginx.MaxmindRetriesTimeout, "maxmind-retries-timeout", time.Second*0, "Maxmind downloading delay between 1st and 2nd attempt, 0s - do not retry to download if something went wrong.")
 
-	flag.Set("logtostderr", "true")
-
+	if err := flag.Set("logtostderr", "true"); err != nil {
+		return false, nil, err
+	}
 	flags.AddGoFlagSet(flag.CommandLine)
-	flags.Parse(os.Args)
+	if err := flags.Parse(os.Args); err != nil {
+		return false, nil, err
+	}
 
 	// Workaround for this issue:
 	// https://github.com/kubernetes/kubernetes/issues/17162
-	flag.CommandLine.Parse([]string{})
+	if err := flag.CommandLine.Parse([]string{}); err != nil {
```

Comment:
This is a pretty old issue, we should probably just remove this line

Reply:
Okay, I also removed `flag.Set("logtostderr", "true")` because this line always returns an error: `no such flag -logtostderr` for all tests in flags_test.go, so I think it seems deprecated